### PR TITLE
CELDEV-272

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-272

### CODE REVIEW

File: src/main/java/com/xpn/xwiki/plugin/lucene/LucenePlugin.java [hunk 1]
```diff
@@ -693,36 +695,17 @@ public synchronized void init(XWikiContext context) {
...
-      maxQueueSize = (int) context.getWiki().ParamAsLong(LucenePlugin.PROP_MAX_QUEUE_SIZE, 1000);
-    } catch (NumberFormatException exp) {
-      LOGGER.warn("Invalid max queue size in configuration.", exp);
-      maxQueueSize = 1000;
-    }
-
-    IndexUpdater indexUpdater = new IndexUpdater(directory, indexingInterval, maxQueueSize, this,
-        context);
-
+    int indexingInterval = 1000 * (int) context.getWiki().ParamAsLong(PROP_INDEXING_INTERVAL, 30);
```
Comment (fpichler) [inline src/main/java/com/xpn/xwiki/plugin/lucene/LucenePlugin.java:698]: maybe just change to long... :-/

File: src/main/java/com/xpn/xwiki/plugin/lucene/LucenePluginApi.java [hunk 2]
```diff
@@ -97,11 +97,11 @@ public int rebuildIndex() {
...
    * @param hqlFilter
-   * @param clearIndex
+   * @param wipeIndex
    * @param onlyNew
    * @return number of documents scheduled for indexing. -1 in case of errors
    */
-  public int startIndex(Collection<String> wikis, String hqlFilter, boolean clearIndex,
+  public int startIndex(Collection<String> wikis, String hqlFilter, boolean wipeIndex,
       boolean onlyNew) {
     if (hasAdminRights()) {
```
Comment (fpichler) [inline src/main/java/com/xpn/xwiki/plugin/lucene/LucenePluginApi.java:106]: please check for superAdmin instead of local admin

**Comment by fpichler** [comment]
looks good. Only small things, after changing please merge.

**Comment by msladek** [comment]
comments adressed in #10

**Pull request merged**